project_profile_counter.py

```diff
@@ -0,0 +1,28 @@
+import json
+import requests
+import collections
+
+SERVER = 'https://tokyo.sdelements.com'
+API_TOKEN = 'f258c47557a3f98f55d4fbd0cb9d8354c86fbb52'
```

Comment:
Ah, obvious comment I missed: you should revoke this API token and write code so that we aren't hard coding tokens.

```
import os
...
SERVER = os.environ['SDE_SERVER']
API_TOKEN = os.environ['SDE_API_TOKEN']
```